test/public/defaults.js

```diff
@@ -308,6 +308,11 @@ module.exports.expectInnerTextTo = async (page, selector, validator) => {
     expect(validator(actualInnerText), `"${actualInnerText}" is invalid with respect of given validator`).to.be.true;
 };
 
+module.exports.expectSelectToBe = async (page, selector, selectedOption) => {
```

Comment:
I would rather name it `expectInputValue`, and the third variable to be `expectedValue`

Reply:
reused other function